src/index.html

```diff
@@ -11,6 +11,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form form_big"
+      data-qa="big">
+      <input
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      class="form form_small"
+      data-qa="small">
+      <input
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```

Comment:
```suggestion
    <form
      class="form form_big"
      data-qa="big"
    >
      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
    </form>

    <form
      class="form form_small"
      data-qa="small"
    >
      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```
Consider fixing indents according to the code style guide